package.json

```diff
@@ -48,6 +49,9 @@
   "scripts": {
     "dev": "nodemon --inspect -w ./lib ./bin/armadietto.js -c ./bin/dev-conf.json",
     "test": "mocha -u bdd-lazy-var/getter  spec/runner.js",
+    "create-user": "NODE_NO_WARNINGS=1 node --experimental-fetch ./bin/create-user.js",
+    "measure": "NODE_NO_WARNINGS=1 node --experimental-fetch ./bin/measure.js",
+    "stress-test": "mocha spec/stress_runner.js",
```

Comment:
The doc update mentions `measure` and `create-user` but not `stress-test`. I'm not sure how each should be used or what the difference is?

Reply:
`stress-test` was written as automated tests, but automated performance tests are hard to make reliable.  That will probably be dropped in the final version of this PR.

`measure` is a manual command-line test.

This was originally written before #90. After #90 is merged, I'll need to do some cleanup to accommodate those changes.